packages/eslint-plugin/tests/rules/ban-ts-comment.test.ts

```diff
@@ -5,14 +5,16 @@ const ruleTester = new RuleTester({
   parser: '@typescript-eslint/parser',
 });
 
-ruleTester.run('ts-ignore', rule, {
+ruleTester.run('ban-ts-comment', rule, {
```

Comment:
This was called `ts-ignore` because it's testing only `ts-ignore`s. That's why the other runs are `ts-check`, `ts-nocheck` etc

(Also no idea why I used backticks for these valids - I noticed this while doing `invalid-void` and facepalmed).